verilog/CST/verilog_matchers.h

```diff
@@ -280,6 +280,16 @@ static const auto AlwaysStatementHasEventControlStar =
 static const auto AlwaysKeyword =
     verible::matcher::MakePathMatcher({L(TK_always)});
 
+// Matches occurrence of the 'always_comb' keyword.
+// This is needed to distinguish between various kAlwaysStatement's.
+static const auto AlwaysCombKeyword =
+    verible::matcher::MakePathMatcher({L(TK_always_comb)});
+
+// Matches occurrence of the 'always_ff' keyword.
+// This is needed to distinguish between various kAlwaysStatement's.
+static const auto AlwaysFFKeyword =
+    verible::matcher::MakePathMatcher({L(TK_always_ff)});
+
```

Comment:
Please add a few unit tests to verilog_matchers_test.cc for these.  This will help us detect/correct future changes faster.

Reply:
Done